### ISSUE TITLE
How to assign a landing page with different template - also maintenance mode bug

### ISSUE THREAD

**Issue by ilu33** [hibbitts-design/grav-skeleton-learn2-with-git-sync#4]
I'm having a hard time to understand how to create and apply a template (I created a theme with inheritance) to the page grav starts up with. Assigning a different template to the start page doesn't work because start is actually the second page and not the first. No matter what I do the login window appears with the navigation already visible.

If I un-publish the start page the login frame vanishes but the navigation is still visible.

If I activate the maintenance plugin the navigation still shows up albeit in a scrambled way. Actually, this must be a bug because if I do the same with the learn2 theme the menu is visible as normal. Anyway, the desired effect in maintenance mode is that you see only the maintenance mode announcement and NOTHING from the site, not even the navigation.

What I'm trying to achieve: 
1. open grav with a landing page that offers a link to login. Alternatively open grav with only the login window visible.
2. show any learn2 content (incl. navigation) only to authorized users. Show nothing to unauthorized users.

Can you please point me in the right direction? Where did you define the navigation part in a way that it can't be hidden no matter what I (or other plugins) do?

**Comment by paulhibbitts** [comment]
Hi @ilu33 , sorry for the delay in getting back to you.

As you have seen, the Learn2 with Git Sync theme is an inherited theme of Learn2 and many of the characteristics of my theme are based on that theme. In general, Learn2 is quite unique as compared to other Grav themes so you may find some techniques not working as expected.

In terms of showing any content of a Learn2 with Git Sync site only to certain users, have you considered using .htaccess for this function (http://www.htaccesstools.com/articles/password-protection/)? As all of my sites are fully public I really don't have experience with user access rights etc. Overtime when this issue has come up I've seen others either recommend the .htaccess route or the Login Plugin (https://github.com/getgrav/grav-plugin-login) along with the Private Plugin(https://github.com/Diyzzuf/grav-plugin-private).

Please let me know if the above is helpful.
Paul

**Comment by ilu33** [comment]
The server is using nginx which does not support .htaccess files (and the nginx developers have valid reasons for this decision). Also the site will have more than just a few users with diverging permissions for parts of the site and managing them in two places - a file and in the user plugin - is not a good idea. 

The private plugin doesn't hide the menu (there are bug reports about it), the maintenance plugin doesn't. And it doesn't seem to be related to theme choice. 

But since the layout part of the menu goes away with your skeleton (so you must have done something differently than the other themes) I thought that maybe there's a way to force the text to go away too.

This is probably a bug, because with Learn2 and maintenance plugin the menu looks right. But if I enable Learn2+git with maintenance plugin the menu css layout gets lost and just the scrambled text remains. Switching back to Learn2 and the menu looks correct again. (English is not my primary language - do you understand what I mean?)
Looking at the css I see that for example this css is missing with with Learn2+git so no background color:
```css
#sidebar ul.topics > li.parent, #sidebar ul.topics > li.active {
    background: #2d353e;
    margin-left: -1rem;
    margin-right: -1rem;
    padding-left: 1rem;
    padding-right: 1rem;
}
```
It seems that theme.css doesn't get loaded or maybe no css at all. 

You fixing this bug won't help me but I'm reporting it anyway. I'd prefer if you sent the text the same way the layout went. :-)

**Comment by paulhibbitts** [comment]
Thanks for the follow-up @ilu33. I've taken a quick look and I suspect it is an issue of Learn2 with Git Sync being an inherited theme (not a child theme) of Learn2 but I am not sure and have not tested that. As I do not understand myself what the issue is I can't provide a fix at this time.

You could try to make a child theme of Learn2, and then customize that theme (using any part of my Learn2 with Git Sync you want). Another option might be Learn4, which uses the Spectre CSS framework but is not an official release and is not currently supported: https://github.com/getgrav/grav-theme-learn4

**Comment by ilu33** [comment]
I managed a workaround by modifying the maintenance plugin template 
```
{% extends 'partials/base.html.twig' %}

{% block sidebar %}
{% endblock %}
...
```
and by providing a customized maintenance.md file. That way the misformatted sidebar stays invisible - which should be the maintenance default anyway. I had to also copy and rename the base template to a maintenance-specific version in order to also hide the burger menu. So problem solved, thank you!
